docs/install-h2o-deepwater.md

```diff
@@ -0,0 +1,80 @@
+# Install H2O Deep Water
+
+To install H2O Deep Water, there are two options available:  
+
+- Use provided pre-compiled files 
+- Build from source. 
+
+Note that compiling from source will provide the latest or specific version of H2O Deep Water.

## H2O Deep Water: Artifacts

You can install H2O Deep Water artifacts using Python or MXNet.

**H2O Python Module**

Install the H2O python module (Wheel file): ``$ sudo pip install h2o*.whl``

**MXNet**

Install MXNet (EGG file): ``$ sudo easy install mxnet-0.7.0-py2.7.egg``
```

Comment:
Hmm.. I have no idea. They should not be there. I don't see those in my editor. 
<img width="1078" alt="screen shot 2017-01-19 at 4 40 15 pm" src="https://cloud.githubusercontent.com/assets/18428990/22131170/1ac299e8-de66-11e6-8beb-312b2e8a16bc.png">